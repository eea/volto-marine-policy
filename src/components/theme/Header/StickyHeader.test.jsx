import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';

import StickyHeader from './StickyHeader';

const mockStore = configureStore();

test('renders a sticky header component', () => {
  const store = mockStore({
    intl: {
      locale: 'en',
      messages: {},
    },
  });

  const component = renderer.create(
    <Provider store={store}>
      <StickyHeader stickyBreakpoint={700} />
    </Provider>,
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
