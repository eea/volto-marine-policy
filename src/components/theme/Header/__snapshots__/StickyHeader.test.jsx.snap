// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a sticky header component 1`] = `
<div
  className=""
/>
`;
